File: Java ASD/SEM 1/Lab 3/src/Task3.java
public class Task3 {
    public static double task3(double x) {
        double y = 0;
        for (int i = 0; i < 6; i++) {
            double term = Math.pow(-1, i + 1) * Math.pow(x, 2 * i) / task31(2 * i - 1);
            y += term;
        }
        return y;
    }
    public static long task31(int n) {
        if (n <= 1) {
            return 1;
        } else return n * task31(n - 1);
    }
}

File: C & C++ Basic Programing/Sem1/Laba4 - Control Statements/task3.c
#include <stdio.h>
#include <math.h>

int main() {
	int n, i;
	double a, sum;
	char cont; 
	do {
        printf("Enter a natural number: ");
        scanf(" %d", &n);
        for (i = 1; i <= n; i++) {
        a = pow(-1, i + 1) / (2 * i - 1);
        sum += a;
	    }
        printf("\na = %lf\n", sum);
        printf("\nDo you want to continue? If YES enter '+', if NO enter '-': ");
        scanf(" %c", &cont);
        while (cont != '+' && cont != '-') {
            printf("Invalid input! Try again:");
            scanf(" %c", &cont);
        }        
    } while (cont == '+');
    printf("\nBye Bye\n");
    return 0;
}